### PR TITLE
added git info when adding --git or -g option to packager:list

### PR DESCRIPTION
**Pull request by happyDemon**
pull request for part of issue #73

### CODE REVIEW

File: src/Commands/ListPackages.php [hunk 1]
```diff
@@ -42,7 +44,59 @@ public function handle()
...
+            $this->renderGitPackages($packages, $packages_path);
+
+            return;
+        }
+
         $headers = ['Package', 'Path'];
         $this->table($headers, $packages);
     }
+
+    protected function renderGitPackages($packages, $packages_path)
```
Comment (Jeroen-G) [inline src/Commands/ListPackages.php]: Make this `private`

File: src/Commands/ListPackages.php [hunk 2]
```diff
@@ -42,7 +44,59 @@ public function handle()
             }
         }
 
+        if ($this->option('git')) {
+            $this->renderGitPackages($packages, $packages_path);
+
+            return;
```
Comment (Jeroen-G) [inline src/Commands/ListPackages.php]: This could probably be `return $this->renderGitPackages(`

File: src/Commands/ListPackages.php [hunk 3]
```diff
@@ -42,7 +44,59 @@ public function handle()
...
+                            $branch = str_replace(["\n", "\r", ' ', '*'], '', $buffer);
+                        }
+                    });
+
+                return [
+                    $package['vendor'],
+                    $package['name'],
+                    $commitDifference,
+                    $branch,
+                ];
```
Comment (Jeroen-G) [inline src/Commands/ListPackages.php]: This is pretty long and cryptic at first glance, so perhaps you could extract it to another function?

Comment (happyDemon) [reply src/Commands/ListPackages.php]: I isolated the the commits difference & branch name retrieval into their own function with better comments. Hope that's enough :)

**Comment by Jeroen-G** [comment]
I merged it so you get the credits, but there was a bug with a variable name, and I refactored it a bit.
Many thanks your for work! 👍

**Pull request merged**